packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -260,4 +260,4 @@ export * from './scene';
 
 export { codegen as codegenExpression, parse as parseExpression } from 'vega-expression';
 
-export { selector as parseSelector } from 'vega-event-selector';
```

Comment:
I think my prev comment was unclear — I had wanted us to be consistent (`parseSelector` and `parseExpression` or `selectorParse` and `expressionParse`). I think we should keep the rename here as `parseSelector`.

Reply:
Done